Jamulus.pro

```diff
@@ -295,12 +295,24 @@ win32 {
     isEmpty(PREFIX) {
         PREFIX = /usr/local
     }
+
     isEmpty(BINDIR) {
         BINDIR = bin
     }
     BINDIR = $$absolute_path($$BINDIR, $$PREFIX)
-    INSTALLS += target
     target.path = $$BINDIR
+    INSTALLS += target
+
+    isEmpty(APPSDIR) {
+        APPSDIR = share/applications
+    }
+    APPSDIR = $$absolute_path($$APPSDIR, $$PREFIX)
+    desktop.path = $$APPSDIR
+    desktop.files = distributions/jamulus.desktop
+    # the .desktop file assumes the binary is called jamulus
```

Comment:
It should be possible to rename the desktop file end with `.in`, put `$$TARGET` in the `Exec` file and then use `QMAKE_SUBSTITUTES += distributions/jamulus.desktop.in`.

See https://martin.rpdev.net/2017/05/10/creating-config-header-files-using-qmake.html or https://github.com/detrout/debian-signon-ui/blob/c03c68965f92285d3d5341b45414473081ed5c1b/src/signon-ui.pro#L105-L109.

Reply:
Ahh! That is... extremely well hidden in the qmake documentation.